File: Selenium/Activities/Activity2.java
package activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity2 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/login-form/");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		driver.findElement(By.id("username")).sendKeys("admin");
		
		driver.findElement(By.id("password")).sendKeys("password");
		
		driver.findElement(By.cssSelector("button.svelte-1pdjkmx")).click();
		
		driver.quit();
		

	}

}

File: Selenium/Activities/activity_2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/login-form/")

    print("Title of the page:", driver.title)

    driver.find_element(By.ID, "username").send_keys("admin")

    driver.find_element(By.ID, "password").send_keys("password")

    driver.find_element(By.CSS_SELECTOR, "button.svelte-1pdjkmx").click()

    driver.quit()